lib/v2/swjtu/jyzpxx.js

```diff
@@ -0,0 +1,77 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+const rootURL = 'https://jiuye.swjtu.edu.cn/career/';
+
+module.exports = async (ctx) => {
+    const resp = await got({
+        method: 'post',
+        url: `${rootURL}/zpxx/search/zpxx/1/30`,
```

Comment:
`rootURL` ended with a slash, `url` also has a slash after `rootURL`. The final URL will contain **duplicated** slashes